content/applications/general/email_communication/email_servers.rst

```diff
@@ -197,26 +197,21 @@ alias in your mail server.
 Set up different dedicated servers for transactional and mass mails
 ===================================================================
 
-Odoo is subject to a :ref:`daily email limit <email_communication/daily_limit_mail>` to prevent abuse. 
-However, if needed, you can use a separate Mail Transfer Agent (MTA) servers for transactional
-e-mails and mass mailings.
-Example: use Odoo's own mail server for transactional e-mails, and Sendgrid, Amazon SES, or Mailgun
-for mass mailings. Another alternative is to use Postmark for transactional e-mails, and Amazon SES
-or Sendgrid for mass mailings.
+In Odoo a separate Mail Transfer Agent (MTA) server can be used for transactional e-mails and mass mailings.
+Example: Use Postmark or SendinBlue for transactional e-mails, and Amazon SES, Mailgun or Sendgrid for mass mailings.
 
 .. note::
    A default outgoing email server is already configured. You should not create an alternative one
    unless you want to use a specific external outgoing email server for technical reasons.
```

Comment:
Replace `You should not create an alternative one unless you want to use a specific external outgoing email server for technical reasons` with `Do not create an alternative one unless a specific external outgoing email server is needed for technical reasons.`